src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj

```diff
@@ -9,15 +9,12 @@
     <Description>Static code analyzer to detect common mistakes or potential issues regarding threading and async coding.</Description>
     <PackageId>Microsoft.VisualStudio.Threading.Analyzers</PackageId>
     <PackageTags>analyzers visualstudio vssdk sdk threading</PackageTags>
-    <Authors>Microsoft</Authors>
-    <Owners>Microsoft, VisualStudioExtensibility</Owners>
-    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageProjectUrl>https://github.com/Microsoft/vs-threading</PackageProjectUrl>
     <PackageReleaseNotes>In the v16.0 release, analyzers require at least C# 7.3 (Visual Studio 2017 Update 7).</PackageReleaseNotes>
 
     <!-- Analyzers should not be added to the lib folder. -->
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <IncludeSymbols>false</IncludeSymbols>
```

Comment:
Why no symbols package?

Reply:
Leaving it on broke the build, since we don't pack pdb files for analyzers. I could also have started including pdb files, but who cares, really, for an analyzer package? No tooling would ever find and use it and customers wouldn't bother either, if the analyzers failed.